client/src/components/add-job-lead-component/index.jsx

```diff
@@ -0,0 +1,284 @@
+  };
+
+  const handleResetInputs = () => {
+    resetInitialState();
+  };
+
+  return (
+    <Container>
+      <H1>Adding a Job Lead</H1>
+      <Body>Input information about the job lead.</Body>
+      {jobLeadData.map((lead) => (
+        <JobLeadContainer key={lead.id}>
+          <H3>Job Lead</H3>
+          <FormControl fullWidth sx={{ m: 1, width: "96%" }}>
+            <InputLabel id="nameLabel">Employer Name</InputLabel>
+            <Select
+              sx={{ textAlign: "left" }}
+              labelId="nameLabel"
+              id={`employer${lead.id}`}
+              value={lead.employer}
+              label="Employer Name"
+              helpertext="*Required"
+              onChange={(e) =>
+                handleInputChange(e.target.value, lead.id, "employer")
+              }
+            >
+              <MenuItem value="name 1">Name 1</MenuItem>
+              <MenuItem value="name 2">Name 2</MenuItem>
+              <MenuItem value="name 3">Name 3</MenuItem>
+            </Select>
+            <FormHelperText>*Required</FormHelperText>
+          </FormControl>
+          <TextField
+            fullWidth
+            sx={{ m: 1, width: "96%" }}
+            id={`title${lead.id}`}
+            value={lead.title}
+            onChange={(e) =>
+              handleInputChange(e.target.value, lead.id, "title")
+            }
+            label="Job Title"
+            helpertext="*Required"
+          />
+          <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+            <InputLabel id="minCompensationLabel">
+              Compensation Minimum
+            </InputLabel>
+            <OutlinedInput
+              id={`minCompensation${lead.id}`}
+              startAdornment={
+                <InputAdornment position="start">$</InputAdornment>
+              }
+              endAdornment={
+                <InputAdornment position="end">/hour</InputAdornment>
+              }
+              label="Compensation Minimum"
+              value={lead.minCompensation}
+              onChange={(e) =>
+                handleInputChange(e.target.value, lead.id, "minCompensation")
+              }
+            />
+          </FormControl>
+          <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+            <InputLabel id="maxCompensationLabel">
+              Compensation Maximum
+            </InputLabel>
+            <OutlinedInput
+              id={`maxCompensation${lead.id}`}
+              startAdornment={
+                <InputAdornment position="start">$</InputAdornment>
+              }
+              endAdornment={
+                <InputAdornment position="end">/hour</InputAdornment>
+              }
+              label="Compensation Maximum"
+              value={lead.maxCompensation}
+              onChange={(e) =>
+                handleInputChange(e.target.value, lead.id, "maxCompensation")
+              }
+            />
+          </FormControl>
+          <TextField
```

Comment:
This should also probably be masked to numbers only

Reply:
I masked this to numbers and decimals if that's okay? I thought employers might want for example 37.5 hours per week.